js/imageViewerAnnotate.js

```diff
@@ -185,9 +192,10 @@ IVEM.insertPreview = function (field, params) {
       $container = $("<div></div>")
         .attr("data-ivem-container", params.container_id)
         .css("position", "relative")
-        .css("padding-top", "30px")
+        .css("margin-top", "50px")
+        .css("margin-bottom", "50px")
         .css("margin-left", "auto")
-        .css("margin-right", "auto");
+        .css("margin-right", "20px");
```

Comment:
why change to 20px? the auto makes it centered.

Reply:
My main concern is having the user on mobile making an annotation near the right edge of screen (see below)

- `.css("margin-right", "20px")`
![image](https://user-images.githubusercontent.com/22246473/144425821-17cc5edf-4bf0-435b-abfc-cb6e34bc7afb.png)

- `.css("margin-right", "auto")`
![image](https://user-images.githubusercontent.com/22246473/144425586-66e12f1a-9480-445e-b74d-1f98e80eb030.png)